php/delivery/class-responsive-breakpoints.php

```diff
@@ -29,7 +29,7 @@ class Responsive_Breakpoints extends Delivery_Feature {
 	 *
 	 * @var string
 	 */
-	protected $settings_slug = 'responsive_breakpoints';
+	protected $settings_slug = 'media_display';
```

Comment:
yup, same as above.
it can be either since the storage name is an alias of the path. By adding the storage name here, it gives a little more clarity of where the data is.

Reply:
Thanks for the explanation